private/numerics/functions/bessel.rkt

```diff
@@ -0,0 +1,367 @@
+#lang racket/base
+
+;; This module defines the bessel functions of integer order
+
+(require racket/math)
+(require racket/fixnum)
+
+(define π pi)
```

Comment:
Can you put these constants in `mechanics/main.rkt` and just use a `(require mechanics)` at the top? We can deal with merge conflicts in `main.rkt` later, but I'm hoping we can organize so that's the only file that'll need changing.